functions.php

```diff
@@ -33,19 +33,17 @@ function check_snippet_existence(){
 	
 	$type = get_post_meta($post->ID, '_bsf_post_type', true);
 	if($type){		
-		add_action( 'wp_head', 'frontend_style' );
 		add_action('wp_enqueue_scripts', 'enque');
 	}
 
 }
-function enque() {
+function enque( ) {

```

Comment:
Add a prefix to function name to make this unique.

Reply:
okay